site/src/pages/WorkspacesPage/WorkspacesTable.tsx

```diff
@@ -189,7 +192,29 @@ export const WorkspacesTable: FC<WorkspacesTableProps> = ({
 								</TableCell>
 
 								<TableCell>
-									{getDisplayWorkspaceTemplateName(workspace)}
+									<span css={{ display: "block" }}>
+										{getDisplayWorkspaceTemplateName(workspace)}
+									</span>
+
+									{dashboard.showOrganizations && (
+										<span
+											css={{
+												display: "block",
+												fontSize: 13,
+												color: theme.palette.text.secondary,
+												lineHeight: 1.5,
+											}}
+										>
+											{/*
+												Only using shorthand version of "organizations" for aesthetics,
+												but because screen readers don't care about aesthetics, we can
+												always display the full text to them
+											*/}
+											org
+											<span css={{ ...visuallyHidden }}>anization</span>:
```

Comment:
are screen readers going to handle this split well though? my gut feeling is no

Reply:
I tried it with the built-in Mac screen reader, but (1) I think I was just assuming that any elements that are semantically inline elements wouldn't affect pronunciation, and (2) the Mac screen reader isn't even the most popular reader by a long shot

Let me change this just to be on the safe side